### PR TITLE
support Exporting expenses against employee default location / department

### PR DESCRIPTION
**Pull request by Sravanksk**
Screenshot 2021-06-11 at 10 23 29 PM

### CODE REVIEW

File: src/app/si/settings/general-mappings/general-mappings.component.html [hunk 1]
```diff
@@ -151,6 +151,14 @@
         </mat-select>
       </mat-form-field>
 
+      <mat-slide-toggle color="primary" class="schedule" formControlName="useDefaultEmployeeLocation" *ngIf="generalSettings.reimbursable_expenses_object === 'EXPENSE_REPORT' && generalSettings.corporate_credit_card_expenses_object !== 'BILL'">
```
Comment (ashwin1111) [inline src/app/si/settings/general-mappings/general-mappings.component.html:154]: Curious on why this condition? If someone uses ER for reimburse and Bill for CCC, are we not going to show this toggle?
`generalSettings.reimbursable_expenses_object === 'EXPENSE_REPORT' && generalSettings.corporate_credit_card_expenses_object !== 'BILL'`

Comment (Sravanksk) [reply src/app/si/settings/general-mappings/general-mappings.component.html:154]: Yes, this is only for - 
Expense Reports in Reimbursable 
ChargeCard Transactions and ExpenseReports in CCC

File: src/app/si/settings/general-mappings/general-mappings.component.html [hunk 2]
```diff
@@ -151,6 +151,14 @@
...
+        <div class="schedule--toggle-text">Use Employee Location</div>
```
Comment (Shwetabhk) [inline src/app/si/settings/general-mappings/general-mappings.component.html:155]: `User Employee's Sage Intacct Location`
add and info icon along with this which says
`Expenses on Sage Intacct will be coded to the Location assigned to the employee on Sage Intacct`

File: src/app/si/settings/general-mappings/general-mappings.component.html [hunk 3]
```diff
@@ -151,6 +151,14 @@
...
+      </mat-slide-toggle>
+
+      <mat-slide-toggle color="primary" class="schedule" formControlName="useDefaultEmployeeDepartment" *ngIf="generalSettings.reimbursable_expenses_object === 'EXPENSE_REPORT' && generalSettings.corporate_credit_card_expenses_object !== 'BILL'">
+        <div class="schedule--toggle-text">Use Employee Department</div>
```
Comment (Shwetabhk) [inline src/app/si/settings/general-mappings/general-mappings.component.html:159]: `User Employee's Sage Intacct Department`
add and info icon along with this which says
`Expenses on Sage Intacct will be coded to the Department assigned to the employee on Sage Intacct`

**Pull request merged**